index.html

```diff
@@ -9,5 +9,6 @@
 </head>
 <body>
     <h1 class="header">Hello Microverse</h1>
+    
```

Comment:
⚠️ REQUIRED CHANGES:

You need to use the README.md template for all Microverse projects. This will make sure that all your projects look professional, and are ready to show off all your skills!  ✨  Please, use this template, and customize it to your project: https://github.com/microverseinc/readme-template .

You have merged to main before this review request, so I can only comment the files that concern this specific request. Kindly only merge to main once you have received an approved code review.

Reply:
Hi @melaniesigrid,
I did what you asked me to do, I hope everything is good.